execution_engine_testing/test_support/src/wasm_test_builder.rs

```diff
@@ -1361,14 +1361,9 @@ where
     }
 
     /// Advances eras by num_eras
-    pub fn advance_eras_by(
-        &mut self,
-        num_eras: u64,
-        reward_items: impl IntoIterator<Item = RewardItem>,
-    ) {
+    pub fn advance_eras_by(&mut self, num_eras: u64) {
```

Comment:
Should we still leave an option to add some rewards while advancing eras? Of course, it would need to work differently, so maybe it's not worth the added complexity? (Not sure what it was being used for.)

Reply:
There's a second part coming (rewards for propagation of finality signatures), but I am yet to work out a design. Note that this is just a function in the test support crate.

Reply:
seems to be primarily used for advancing past initial eras pre-populated at genesis in tests